Geometry/GEMGeometry/src/GEMEtaPartitionSpecs.cc

```diff
@@ -13,14 +13,17 @@ GEMEtaPartitionSpecs::GEMEtaPartitionSpecs(SubDetector rss, const std::string& n
     float r0 = h * (B + b) / (B - b);
     float striplength = h * 2;
     float strips = _p[3];
-    float pitch = (b + B) / strips;
+    float dphi = _p[5] * TMath::Pi() / 180.;
+    float phiPitch = dphi / strips;
+
     int nstrip = static_cast<int>(strips);
-    _top = new TrapezoidalStripTopology(nstrip, pitch, striplength, r0);
+    _top = new GEMStripTopology(nstrip, phiPitch, striplength, r0);
```

Comment:
possibly outside the scope of this PR, but it would be better to use smart pointers

Reply:
I've fixed this PR as your comments, but I can't understand how to use smart pointers for this.